task7/task7.3/storage.js

```diff
@@ -0,0 +1,81 @@
+export {Storage};
+class Storage {
+    _map;
+    constructor() {
+        this._map = new Map();
+    }
+    add(obj) {
+        if (obj) { //check for empty obj
+            this._map.set(this._map.size.toString(), obj);
+        }
```

Comment:
index generation will break on delete (15 added, 1 deleted, next added tries to add index 14, which already exists)